src/Template/Element/index/bulk_actions/form_end.ctp

```diff
@@ -0,0 +1,30 @@
+<?php
+if (empty($bulkActions)) {
+    return;
+}
+
+$this->Form->templates([
+    'submitContainer' => '{{content}}',
+]);
+
+$submitButton = $this->Form->submit(__d('crud', 'Apply'), [
+    'class' => 'btn btn-success btn-bulk-apply',
+    'div' => false,
+    'name' => '_bulk',
+]);
+$this->Form->templates([
+    'inputContainer' => '<div class="form-group bulk-action-submit {{required}}">{{content}}{{help}}</div>',
+    'select' => '<div class="col-sm-10"><select name="{{name}}"{{attrs}}>{{content}}</select>' . $submitButton . '</div>',
+]);
+
+echo $this->Form->input('action', [
```

Comment:
Instead of the `$this->Form->templates()` call above you can pass the templates array using `templates` key to this Form->input() call.